airbyte-cdk/bulk/core/load/src/test/kotlin/io/airbyte/cdk/load/data/json/AirbyteSchemaTypeToJsonSchemaTest.kt

```diff
@@ -4,145 +4,151 @@
-        Assertions.assertEquals(ofType("number", true), propsOut.get("height"))
-
-        val friends = JsonNodeFactory.instance.objectNode()
-        friends.put("type", "array").replace("items", ofType("string", true))
-        Assertions.assertEquals(ofNullable(friends), propsOut.get("friends"))
-
-        val address = JsonNodeFactory.instance.objectNode()
-        val addressProps = address.put("type", "object").putObject("properties")
-        addressProps.replace("street", ofType("string", true))
-        addressProps.replace("city", ofType("string", true))
-        Assertions.assertEquals(ofNullable(address), propsOut.get("address"))
-
-        Assertions.assertEquals(ofType("null", true), propsOut.get("null_field"))
-
-        val nullableUnion = JsonNodeFactory.instance.objectNode()
-        nullableUnion
-            .putArray("oneOf")
-            .add(ofType("string", false))
-            .add(ofType("integer", false))
-            .add(ofType("null", false))
-        Assertions.assertEquals(nullableUnion, propsOut.get("nullable_union"))
-
-        val nonnullableUnion = JsonNodeFactory.instance.objectNode()
-        nonnullableUnion
-            .putArray("oneOf")
-            .add(ofType("string", false))
-            .add(ofType("integer", false))
-        Assertions.assertEquals(nonnullableUnion, propsOut.get("nonnullable_union"))
-
-        Assertions.assertEquals(ofType("string", true), propsOut.get("combined_null"))
-
-        val combinedDenormed = JsonNodeFactory.instance.objectNode()
-        val cdObj = ofType("object", false)
-        cdObj.putObject("properties").replace("name", ofType("string", true))
-        combinedDenormed
-            .putArray("oneOf")
-            .add(ofType("string", false))
-            .add(cdObj)
-            .add(ofType("null", false))
-        Assertions.assertEquals(combinedDenormed, propsOut.get("combined_denormalized"))
-
-        val unionArrayOut = JsonNodeFactory.instance.objectNode()
-        unionArrayOut
-            .put("type", "array")
-            .putObject("items")
-            .putArray("oneOf")
-            .add(ofType("string", false))
-            .add(ofType("integer", false))
-        Assertions.assertEquals(ofNullable(unionArrayOut), propsOut.get("union_array"))
-
-        val timeTypeFieldNames =
-            listOf("time", "timestamp", "time_without_timezone", "timestamp_without_timezone")
-        timeTypeFieldNames.forEach { fieldName ->
-            val expected = props.get(fieldName) as ObjectNode
-            if (listOf("date-time", "time").contains(expected.get("format").asText())) {
-                val formatName = expected.get("format").asText().replace("date-time", "timestamp")
-                if (!expected.has("airbyte_type")) {
-                    expected.put("airbyte_type", "${formatName}_with_timezone")
+    // A json of every supported type
+    private val airbyteType =
```

Comment:
It was simpler just to rewrite this. It's not exhaustive, but I think this is just used for testing anyway?